src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="big-bar"
+      data-qa="big"
+    >
+      <label class="label">
+        <img
+          src="images/Search.svg"
+          alt="icon"
+          class="icon-big"
```

Comment:
fix classes according to the checklist <img width="1058" alt="Screenshot 2022-11-22 at 19 16 56" src="https://user-images.githubusercontent.com/104769168/203379272-0a99f4d4-8a2b-4629-8af1-1e2034877ade.png">